src/modules/users/user-list/users.js

```diff
@@ -65,34 +64,35 @@
             label: "Delete",
             class: "btn-danger custom-class",
             actionFn: function() {
-                userStore.deleteUser(vm.deleteUser.username)
+                userStore.deleteUser(vm.userToDelete.username)
                     .then(function(data) {
-                        vm.showDeleteUserModal = false;
+                        vm.userToDelete.showDeleteUserModal = false;
                         userStore.getUserList()
                             .then(function(data) {
                                 if (data !== null) {
                                     $rootScope.$broadcast("UpdatedUserList", data);
                                 }
-                                Notifications.message("success", "", vm.deleteUser.username + " deleted Successfully.");
+                                Notifications.message("success", "", vm.userToDelete.username + " deleted successfully.");
                             });
 
                     }).catch(function(e) {
-                        vm.showDeleteUserModal = false;
-                        Notifications.message("danger", "", "Error deleting " + vm.deleteUser.username);
+                        vm.userToDelete.showDeleteUserModal = false;
+                        Notifications.message("danger", "", "Error in deleting " + vm.userToDelete.username);
                     });
 
             }
 
         }];
 
-        function deleteUser(username) {
-            vm.showDeleteUserModal = true;
-            vm.deleteUser = {};
-            vm.deleteUser.username = username;
+        function deleteUser(user) {
+
```

Comment:
extra space.

Reply:
@cloudbehl can you point me about which extra space are you referring here?

Reply:
https://github.com/Tendrl/ui/pull/1077/files#diff-f6d02eab6078fb37eb753f68ead47071R88

Reply:
@cloudbehl That I have added intentionally for better readability.

Reply:
We were not using the same aesthetic in the other functions. I think we can remove it but I am fine anyway.